posthog/api/capture.py

```diff
@@ -206,10 +206,14 @@ def get_event(request):
 
         if is_ee_enabled():
             log_topics = [KAFKA_EVENTS_WAL]
-            if settings.PLUGIN_SERVER_INGESTION_HANDOFF and team.organization_id in getattr(
+            # TODO: remove team.organization_id in ... for full rollout of Plugins on EE/Cloud
+            if settings.PLUGIN_SERVER_INGESTION and team.organization_id in getattr(
                 settings, "PLUGINS_CLOUD_WHITELISTED_ORG_IDS", []
             ):
                 log_topics.append(KAFKA_EVENTS_INGESTION_HANDOFF)
+                statsd.Counter(
+                    "%s_posthog_cloud_plugin_server_ingestion_handoff" % (settings.STATSD_PREFIX,)
```

Comment:
we can probably just name this `"%s_posthog_cloud_plugin_server_ingestion"` since after handoff is complete it will be counting plugin server ingestions